gmplot/gmplot.py

```diff
@@ -268,10 +271,14 @@ def write_map(self,  f):
         f.write('\t\tvar myOptions = {\n')
         f.write('\t\t\tzoom: %d,\n' % (self.zoom))
         f.write('\t\t\tcenter: centerlatlng,\n')
-        f.write('\t\t\tmapTypeId: google.maps.MapTypeId.ROADMAP\n')
+        f.write('\t\t\tmapTypeId: google.maps.MapTypeId.%s,\n' % self.map_type)
+        if self.add_scale:
+            f.write('\t\t\tscaleControl: true\n')
         f.write('\t\t};\n')
         f.write(
             '\t\tvar map = new google.maps.Map(document.getElementById("map_canvas"), myOptions);\n')
+        if self.tilt is not None:
+            f.write('\t\tmap.setTilt(%d)\n' % self.tilt)
```

Comment:
We can avoid the extra `setTilt()` call and directly set `tilt` like we've done for `scaleControl`.